### ISSUE TITLE
ERROR: Library not found: libGL.so

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/osk-sdl#58]
I get that when I test it with `lg-mako`, possibly this also happened in #57.
osk-sdl restarts all the time and exists with `libGL.so` not found.

```
commandline read: osk-sdl
commandline read: -n
commandline read: root
commandline read: -d
commandline read: /dev/mapper/mmcblk0p21p2
commandline read: -c
commandline read: /etc/osk.conf
commandline read: -v
INFO: Disabling linux input
(*) Direct/Thread: Started 'SigHandler' (5891) [CRITICAL - OTHER/0] <85316>...

   ~~~~~~~~~~~~~~~~~~~~~~~~~| DirectFB 1.7.7 |~~~~~~~~~~~~~~~~~~~~~~~~~~
        (c) 2012-2015  DirectFB integrated media GmbH
        (c) 2001-2015  The world wide DirectFB Open Source Community
        (c) 2000-2004  Convergence (integrated media) GmbH
      ----------------------------------------------------------------

(*) DirectFB/Core: Single Application Core. (2017-09-15 21:32)
(*) Direct/Memcpy: Using libc memcpy()
(*) Direct/Thread: Started 'Fusion Dispatch' (5892) [MESSAGING - OTHER/0] <85316>...
(*) Direct/Thread: Started 'VT Switcher' (5894) [CRITICAL - OTHER/0] <85316>...
(*) Direct/Thread: Started 'VT Flusher' (5895) [DEFAULT - OTHER/0] <85316>...
(*) DirectFB/FBDev: Found 'msmfb44_80601' (ID 0) with frame buffer at 0x8231e000, 11520k (MMIO 0x00000000, 0k)
(*) Direct/Modules: suppress module 'linux_input'
(*) Direct/Thread: Started 'Keyboard Input' (5896) [INPUT - OTHER/0] <85316>...
(*) DirectFB/Input: Keyboard 0.9 (directfb.org)
(*) DirectFB/Input: Hot-plug detection enabled with Input Hub Driver
(*) Direct/Thread: Started 'tslib Input' (5897) [INPUT - OTHER/0] <85316>...
(*) DirectFB/Input: tslib touchscreen 0 0.1 (tslib)
(*) DirectFB/Graphics: Generic Software Rasterizer 0.7 (directfb.org)
(*) DirectFB/Core/WM: Default 0.3 (directfb.org)
(*) Direct/Thread: Started 'Genefx' (5898) [DEFAULT - OTHER/0] <85316>...
(*) FBDev/Mode: Setting 768x1280 RGB16
(*) FBDev/Mode: Switched to 768x1280 (virtual 768x1280) at 16 bit (RGB16), pitch 1536
(*) FBDev/Mode: Setting 768x1280 RGB16
(*) FBDev/Mode: Switched to 768x1280 (virtual 768x1280) at 16 bit (RGB16), pitch 1536
INFO: DirectFB Device Information
INFO: ===========================
INFO: Name:           Software Rasterizer
INFO: Vendor:         Generic
INFO: Driver Name:    Software Driver
INFO: Driver Vendor:  directfb.org
INFO: Driver Version: 0.7
INFO: Video memoory:  11796480
INFO: Blitting flags:
INFO: Drawing flags:
INFO: Acceleration flags:
ERROR: Library not found: libGL.so
INFO: Keyboard 0 - Keyboard
INFO: Mode 768 x 1280 Added
ERROR: Mode 768 x 1280 not available: f
ERROR: Mode 768 x 1280 not available: f
INFO: Mode 768 x 1280 Added
ERROR: Mode 768 x 1280 not available: f
 (!!!)  *** UNIMPLEMENTED [fusion_get_fusionee_pid] *** [fusion.c:4147]
(*) FBDev/Mode: Setting 768x1280 RGB16
(*) FBDev/Mode: Switched to 768x1280 (virtual 768x1280) at 16 bit (RGB16), pitch 1536
ERROR: Library not found: libGL.so
```

**Comment by craftyguy** [comment]
This should have been fixed in SDL 2.0.6 ([see here](https://bugzilla.libsdl.org/show_bug.cgi?id=3783)). Make sure that it's being installed from Alpine repo: https://pkgs.alpinelinux.org/package/edge/main/armhf/sdl2

**Comment by ollieparanoid** [comment]
I still had `sdl2-9999` installed (which we forked temporarily). Deleted it with:
`sudo rm -v ~/.local/var/pmbootstrap/packages/*/sdl2*`
